### ISSUE TITLE
Add minimal PSP policy

### ISSUE THREAD

**Issue by grampelberg** [linkerd/website#94]
- [x] Add minimum possible PSP to security section (see #100)

The minimum PSP for linkerd:

```yaml
apiVersion: policy/v1beta1
kind: PodSecurityPolicy
metadata:
  name: default
  annotations:
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default'
    apparmor.security.beta.kubernetes.io/allowedProfileNames: 'runtime/default'
    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default'
    apparmor.security.beta.kubernetes.io/defaultProfileName:  'runtime/default'
spec:
  privileged: false
  # Required to prevent escalations to root.
  allowPrivilegeEscalation: false
  allowedCapabilities:
    # The initContainer for LD2 requires this capability to operate. It is
    # unsafe and *should* be removed.
    - NET_ADMIN
    - NET_RAW
  requiredDropCapabilities:
    - SETPCAP
    - MKNOD
    - AUDIT_WRITE
    - CHOWN
    - DAC_OVERRIDE
    - FOWNER
    - FSETID
    - KILL
    - SETGID
    - SETUID
    - NET_BIND_SERVICE
    - SYS_CHROOT
    - SETFCAP
  # Allow core volume types.
  volumes:
    - 'configMap'
    - 'emptyDir'
    - 'projected'
    - 'secret'
    - 'downwardAPI'
    # Assume that persistentVolumes set up by the cluster admin are safe to use.
    - 'persistentVolumeClaim'
  hostNetwork: false
  hostIPC: false
  hostPID: false
  runAsUser:
    rule: 'RunAsAny'
    # The initContainer for LD2 requires running as root to install the
    # iptables rules. This *should* be set.
    # rule: 'MustRunAsNonRoot'
  seLinux:
    # This policy assumes the nodes are using AppArmor rather than SELinux.
    rule: 'RunAsAny'
  supplementalGroups:
    rule: 'MustRunAs'
    ranges:
      # Forbid adding the root group.
      - min: 1
        max: 65535
  fsGroup:
    rule: 'MustRunAs'
    ranges:
      # Forbid adding the root group.
      - min: 1
        max: 65535
  readOnlyRootFilesystem: false
```

**Comment by ihcsim** [comment]
@grampelberg I updated the control plane PSP to (mostly) match the one described in this issue, in https://github.com/linkerd/linkerd2/pull/2969. I didn't add the annotations though, and use `requiredDropCapabilities: All`. LMK if we can close this issue.

**Comment by grampelberg** [comment]
@ihcsim we still need to document what the minimal policy is for the data plane. Did you sneak that in with a website update?

**Comment by ihcsim** [comment]
I'm gonna add this task to the 2.4 Release pile. Now that `linkerd install` creates the control plane PSP by default, it makes sense for us to update the doc to include the minimal PSP for the data plane.

**Comment by grampelberg** [comment]
Excellent!

**Comment by ihcsim** [comment]
Fixed by #392.
